makefile

```diff
@@ -58,7 +58,7 @@ docker: build/containerbuddy_build consul etcd
 
 # top-level target for vendoring our packages: godep restore requires
 # being in the package directory so we have to run this for each package
-vendor: containerbuddy/vendor
+vendor: backends/vendor config/vendor core/vendor discovery/vendor services/vendor utils/vendor telemetry/vendor
```

Comment:
I'm pretty sure we're abusing godep here - I have never seen any project contain several godep manifests and vendor folders in sub modules - I don't this will scale well.

Maybe we can just play with the mount locations and move vendor/godeps back to the top-level?